### ISSUE TITLE
Clarify the choice of Q₀

### ISSUE THREAD

**Issue by kavanase** [WMD-group/CarrierCapture.jl#22]
Flagged by discussions with @kantaimperial; the logic behind the choice of Q₀ in the examples is not really clear. 

From our discussions before @xw-w, this should correspond to the central datapoint used for e-ph coupling right? Can we make this clear with a comment/note in the example tutorial?
![image](https://github.com/user-attachments/assets/717010f6-f462-4137-86f1-42b8ec048b2e)
![image](https://github.com/user-attachments/assets/b695cb8a-0db0-4044-97f0-9bba409a4393)

Also, in the example case given in the notebook, `disp_004` is used in both cases for hole and electron capture e-ph coupling (as we usually do the two together), but then different Q₀ values (`0.4*delQ` and `1.4*delQ`) are used? Is this correct or a typo?

**Comment by xw-w** [comment]
Thanks for pointing it out and I've corrected the typo. The choice of Q₀ is not unique as long as it meets the criterion I noted in the tutorial (i.e. a single-particle defect level should be well defined and be in the bulk band gap) and you may want to do your own tests for your system

**Comment by kavanase** [comment]
Yes but in the notebook it's not obvious to most people that the configuration chosen for e-ph coupling (`disp_004` in this case) is the same as Q₀, so I think this should be clearly stated

**Comment by xw-w** [comment]
Sorry but I thought I wrote in the notebook that "The reference WAVECAR used for WSWQ calculations corresponds to {Q_0}. "? Please feel free to edit the example if you think anywhere is not clear!  :)

**Comment by kavanase** [comment]
Ok yeah but the connection to the later parameters and what exactly Q_0 is was not very clear I think. 
I've now updated this in a43943f8c3cb9059e1dff4e3d9e715541804578b.
Also btw, I think there was a new typo added in your last changes -- you used `0.6*delQ` but should it be `0.4*delQ` for `disp_004`? I changed it to `0.4*delQ`, but would be good if you can confirm this is correct!
